src/components/v5/common/ActionSidebar/partials/forms/CreateDecisionForm/consts.ts

```diff
@@ -6,11 +6,11 @@ export const validationSchema = yup
     title: yup
       .string()
       .trim()
-      .required(() => 'Please enter a title'),
+      .required(() => 'Please enter a title.'),
     createdIn: yup.string().defined(),
     description: yup
       .string()
-      .notOneOf(['<p></p>'], () => 'Please enter a description')
+      .notOneOf(['<p></p>'], () => 'Please enter a description.')
```

Comment:
Shouldn't those messages come from react-intl?

Reply:
Yes, this is actually quite a big one, with other places I have noticed the same. It is probably best done in another PR.